content/en/blog/2023/end-user-discussions-03.md

```diff
@@ -0,0 +1,273 @@
+
+## What we talked about
+
+Sampling and collector capabilities continue to be topics of interest, along
+with questions about instrumentation and adoption.
+
+## Discussion Highlights
+
+Below is the summary of this month's discussions.
+
+### OpenTelemetry Collector
+
+#### 1 - Losing gRPC with Azure App Services
+
+**Q:** When looking at the hosting models in Azure for the OTel Collector, only
+HTTP is supported (for running in Azure App Service). What are the risks
+associated with losing gRPC capability?
+
+**A:** If HTTP2 is supported in Azure, gRPC might work there, since gRPC is HTTP
+under the hood with extra complications built on top of HTTP2. One suggestion is
+to follow up with Microsoft about gRPC support, as it may have very long-running
+connections.
+
+#### 2 - Uptime monitoring/synthetics
+
+**Q:** Does the OTel Collector have the capability to do uptime monitoring/
+synthetics? If not, are there any plans to work towards such a thing?
+
+**A:** The
+[health check](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md)
+might be a helpful reference. Also check out the
+[http check receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md).
+
+#### 3 - Collector distributions
+
+**Q:** Should I use a vendor distribution versus the community collector
+distribution?
+
+**A:** Each vendor distribution will come with customizations, whereas the
+community Collector distribution will include everything: receivers and
+exporters. If you need the flexibility, then you should use the OTel Collector
+distro.
+
+#### 4 - Rate limiting on receivers
+
+**Q:** Each vendor distribution will come with customizations, whereas the
+community Collector distribution will include everything: receivers and
+exporters. If you need the flexibility, then you should use the OTel Collector
+distro.
+
+**More context**: How do I rate-limit in a situation where I have collectors for
+traces, metrics, and logs, and I’m receiving traffic from more than 100
+individual apps? If I have even one customer who is generating heavy traffic, it
+might impact the overall health of my collectors.
+
+**A:** Use a reverse proxy. Something to note is that once the data is inside
+the collector, the data is already being deserialized, and you’ve already
+started firehosing the collector, so it’s a bit late to rate limit at that
+point. One approach might be to add additional headers when you configure your
+SDKs that contain the additional information, which would help with load
+balancing.
+
+#### 5 - Connectors
+
+**Q:** What are connectors?
+
+**A:**
+[Read about connectors here](https://o11y.news/2023-03-13/#opentelemetry-connectors).
+
+#### 6 - Definitions of upstream, downstream, and distro
+
+**Q:** What is upstream? Downstream? Distro?
+
+**A:** Check out
+[this documentation](https://opentelemetry.io/ecosystem/vendors/) and this
```

Comment:
Again, would it be helpful to include high-level definitions here before referring readers to the links?